src/CarrierCode/CarrierCode.php

```diff
@@ -0,0 +1,146 @@
+        self::LATVIJAS_PASTS            => 'https://www.pasts.lv/en/Category/Tracking_of_Postal_Items/',
+        self::LIETUVOS_PAŠTAS           => 'https://old.post.lt/en/help/parcel-search',
+        self::POSTLUXEMBOURG            => 'https://www.post.lu/en/particuliers/colis-courrier/track-and-trace#/result',
+        self::POS_MALAYSIA              => 'https://parcelsapp.com/en/tracking/%s',
+        self::MALTA_POST                => 'https://www.maltapost.com/tracking#/tracking',
+        self::MEXICO                    => 'https://www.correosdemexico.gob.mx/SSLServicios/SeguimientoEnvio/seguimientoportal2.aspx?guia=%s',
+        self::POSTNL                    => 'https://postnl.post/',
+        self::NEW_ZEALAND_POST          => 'https://www.nzpost.co.nz/tools/tracking',
+        self::NIGERIAN_POSTAL_SERVICE   => 'https://www.nipost.gov.ng/Track_Trace',
+        self::POSTEN_NORGE              => 'https://sporing.posten.no/sporing/%s?lang=no',
+        self::BRING                     => 'https://sporing.posten.no/sporing/%s?lang=no',
+        self::PHLPOST                   => 'https://www.phlpost.gov.ph/',
+        self::POCZTA_POLSKA             => 'https://emonitoring.poczta-polska.pl/',
+        self::CTT                       => 'https://www.ctt.pt/feapl_2/app/open/objectSearch/objectSearch.jspx?lang=def',
+        self::POȘTA_ROMÂNĂ              => 'https://www.posta-romana.ro/track-trace.html',
+        self::RUSSIA_POST               => 'https://www.pochta.ru/tracking#%s',
+        self::SINGAPORE_POST            => 'https://www.singpost.com/send-receive/receive-mail-parcel',
+        self::SLOVENSKÁ_POŠTA           => 'https://tandt.posta.sk/',
+        self::POŠTA_SLOVENIJE           => 'https://sledenje.posta.si/',
+        self::SOUTH_AFRICAN_POST_OFFICE => 'http://globaltracktrace.ptc.post/gtt.web/',
+        self::CORREOS                   => 'https://www.correos.es/es/es/herramientas/localizador/envios/detalle?tracking-number=%s',
+        self::POSTNORD                  => 'https://www.postnord.se/vara-verktyg/spara-brev-paket-och-pall?shipmentId=%s',
+        self::SWISS_POST                => 'https://service.post.ch/ekp-web/ui/list',
+        self::CHUNGHWA_POST             => 'https://postserv.post.gov.tw/pstmail/main_mail.html?targetTxn=%s',
+        self::THAILAND_POST             => 'https://track.thailandpost.co.th/',
+        self::PTT                       => 'https://gonderitakip.ptt.gov.tr/Track/Result',
+        self::ROYAL_MAIL                => 'https://www.royalmail.com/track-your-item#/tracking-results/%s',
+        self::PARCELFORCE               => 'https://www.parcelforce.com/contact-us',
+        self::CORREO_URUGUAYO           => 'https://www.correo.com.uy/seguimientodeenvios',
+        self::USPS                      => 'https://tools.usps.com/go/TrackConfirmAction?tRef=fullpage&tLc=2&text28777=&tLabels=%s%2C&tABt=false',
+    ];
+
+    public function __construct(private string $carrierCode)
```

Comment:
As I mentioned in the stand-up: we have to have a way to find the final mile carrier based on the data we have. The main clue we have is the recipient country code. Based on it we can find the information for the final mile carrier (we will also need the name of the carrier too in this case). I suggest the following changes:

1. Lets name the class FinalMileCarrier
2. The constructor can still accept a code, but the country code instead.
3. The mapping above can stay but in a modified state. We need to map `countryCode -> map[url, name]`

Reply:
Small addition, I looked into the countries that we will be supporting. Some countries have multiple carriers listed, I think it will be better if we map `countryCode -> [name, name]` and then map `name => url`. What do you think @yoan-myparcel

Reply:
We will then have 2 final mile carriers after using the library though so how will we know which to use?

Reply:
I expected Lars to filter the list. Lets select the more obvious choices ourselves (I will mark my suggestions to the right one in green). Meanwhile I will persuade Lars to check the list

Reply:
@zwaans done! (the shade of green doesn't matter)